Makefile

```
@@ -0,0 +1,16 @@
+OUTDIR = bin/$(ARCH)
+CFLAGS += -DUNICODE -municode -std=c11 -fgnu89-inline -D_WIN32_WINNT=0x0600
```

```suggestion
CFLAGS += -DUNICODE -municode -std=c11 -fgnu89-inline -D_WIN32_WINNT=0x0600 -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fcf-protection
```

not sure if those work everywhere but they should be used if possible

---

Those (and many more) are already defined by `redhat-rpm-config` package. It may be necessary to add `%set_build_flags` in the rpm spec file to make it effective (but in several cases it is done implicitly already, and I think with rpm-build new enough it is simply the default). This is yet another reason to _not_ override CFLAGS etc (but only modify existing variable, if necessary) @DemiMarie .